sys/docker/alpine-phpfpm/start.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
 cd /application
-/usr/local/bin/composer dump-env prod
+
```

Comment:
I wouldn't comment it, since it's used on prod that would not dump the env file. I'd rather use an if based on the environment

Reply:
added condition!